aif360/sklearn/detectors/detectors.py

```diff
@@ -1,10 +1,76 @@
-from typing import Union
-
+from aif360.detectors.ot_detector import ot_bias_scan
 from aif360.detectors import bias_scan
 from aif360.detectors.mdss.ScoringFunctions import ScoringFunction
 
+from typing import Union
 import pandas as pd
+import numpy as np
+
+def ot_bias_scan(
+    y_true: Union[pd.Series, str],
+    y_pred: Union[pd.Series, pd.DataFrame, str],
+    sensitive_attribute: Union[pd.Series, str] = None,
+    X: pd.DataFrame = None,
+    pos_label: Union[str, float] = None,
+    overpredicted: bool = True,
+    scoring: str = "Optimal Transport",
+    num_iters: int = 100,
+    penalty: float = 1e-17,
+    mode: str = "ordinal",
+    **kwargs,
```

Comment:
same as other comment -- no need to include unused args -- however, in this case, it's good to have the same ordering as `bias_scan`:
`X, y_true, y_pred, prot_attr, pos_label, ...`

Reply:
I see, so pass the features is optional. In that case it's fine as is although I don't really see a scenario where it's easier to pass `X` and strings for `y_true`, `y_pred`, etc. since you're not actually using the features in the algorithm. `prot_attr` is often a str because in `aif360/sklearn/` we store them in the index of the `pd.Series` or you could pass an array directly but I think maybe it's more confusing to combine features and labels and call it `X`.

Do you feel strongly about keeping the `X`/`data` option or can we just drop it and only pass arrays/Series?

Reply:
dropped the `X` option in the new pull request